### ISSUE TITLE
Unexpected footer "We've started the requested process in a new window."

### ISSUE THREAD

**Issue by svenfoo** [pretix/pretix#3183]
### Problem and impact

We are running a self-hosted pretix instance using the **pretix/standalone:stable** Docker image pulled from **docker.io**.

Every page in the accounts section https://tickets.foo.bar/organisation/account/ shows an unexpected footer saying the following:

```
We've started the requested process in a new window.

If you do not see the new window, we can help you launch it again.
 -------------------
| Open window again |
 -------------------
Once the process in the new window has been completed, you can continue here. 
```

We don't quite understand what this footer is trying to tell us and we believe that it is displayed inadvertently. Is this a bug, perhaps even a known problem, or a misunderstanding on our side?

### Expected behaviour

We would expect not to see this footer. It does not seem to have any purpose.

### Screenshots

![Screenshot from 2023-03-26 14-59-28](https://user-images.githubusercontent.com/3492822/227777354-61f5318b-9b45-49f1-a0db-553a9408032c.png)

### Browser (software, desktop or mobile?) and version

Firefox 110 Desktop

### Operating system, dependency versions

Linux

### Version

https://hub.docker.com/layers/pretix/standalone/stable/images/sha256-d2c18eb5141a6814936a07eccd10d09973eae79a7a70695269d438e163d9f611

**Comment by raphaelm** [comment]
Pretty sure you did non run the full update procedure :)
https://docs.pretix.eu/en/latest/admin/installation/docker_smallscale.html#updates

**Comment by svenfoo** [comment]
Indeed, you are correct. Sorry for having missed that and for wasting your time with this bug report. Thanks for the quick help.
